tests/lib/rules/jsx-indent.js

```diff
@@ -1038,7 +1058,10 @@ const Component = () => (
       '}'
     ].join('\n'),
     options: [2],
-    errors: [{message: 'Expected indentation of 2 space characters but found 9.'}]
+    errors: [
+      {message: 'Expected indentation of 2 space characters but found 9.'},
+      {message: 'Expected indentation of 2 space characters but found 9.'}
+    ]
```

Comment:
Tests were failing with

```
AssertionError [ERR_ASSERTION]: Should have 1 error but had 2: 

[
  {
    ruleId: 'jsx-indent',
    severity: 1,
    message: 'Expected indentation of 2 space characters but found 9.',
    line: 2,
    column: 3,
    nodeType: 'ReturnStatement',
    endLine: 4,
    endColumn: 17,
    fix: {range: [Array], text: '  '}
  },
  {
    ruleId: 'jsx-indent',
    severity: 1,
    message: 'Expected indentation of 2 space characters but found 9.',
    line: 4,
    column: 10,
    nodeType: 'JSXClosingElement',
    endLine: 4,
    endColumn: 16,
    fix: {range: [Array], text: '  '}
  }
];

+ expected - actual

-2
+1
```

Looks like https://github.com/yannickcr/eslint-plugin-react/pull/1956 added handling for a few similar cases like I did.